app/screens/channel/index.tsx

```diff
@@ -20,10 +20,6 @@ type EnhanceProps = WithDatabaseArgs & {
 
 const enhanced = withObservables([], ({database, serverUrl}: EnhanceProps) => {
     const channelId = observeCurrentChannelId(database);
-    const isCallsPluginEnabled = observeCallsConfig(serverUrl).pipe(
```

Comment:
Removing this; it was a mistake to add it -- it doesn't matter if the calls plugin is enabled for the current server, the user could still be on a call from a different server.